### PR TITLE
tikv: add configuration docs for I/O rate limiter

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
+ add configuration docs for I/O rate limiter
+ Update default value for RocksDB soft-pending-compaction-bytes-limit

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5729 https://github.com/pingcap/docs/pull/5748
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by yiwu-arbug** [comment]
@TomShawn can you add @tabokie also as reviewer? Thanks.

**Comment by TomShawn** [comment]
@tabokie PTAL, thanks!

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -298,6 +298,21 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
 + 默认值：系统总内存大小的 45%
 + 单位：KB|MB|GB
 
+## storage.io-rate-limit
+
+I/O rate limiter 相关的配置项。
+
+### `max-bytes-per-sec`
+
++ 限制服务器每秒从磁盘读取数据或写入数据的最大 I/O 字节数，I/O 类型由下面的 `mode` 配置项决定。达到该限制后，TiKV 倾向于后台操作为前台操作节流。该配置项值应设为磁盘的最佳 I/O 带宽，例如云盘厂商指定的最大 I/O 带宽。
```
Comment (tabokie) [inline tikv-configuration-file.md:307]:
```suggestion
+ 限制服务器每秒从磁盘读取数据或写入数据的最大 I/O 字节数，I/O 类型由下面的 `mode` 配置项决定。达到该限制后，TiKV 倾向于放缓后台操作为前台操作节流。该配置项值应设为磁盘的最佳 I/O 带宽，例如云盘厂商指定的最大 I/O 带宽。
```

**Review by tabokie — APPROVED** [review]
LGTM

**Review by Joyinqin — APPROVED** [review]
/lgtm

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -298,6 +298,21 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
...
+I/O rate limiter 相关的配置项。
+
+### `max-bytes-per-sec`
+
++ 限制服务器每秒从磁盘读取数据或写入数据的最大 I/O 字节数，I/O 类型由下面的 `mode` 配置项决定。达到该限制后，TiKV 倾向于放缓后台操作为前台操作节流。该配置项值应设为磁盘的最佳 I/O 带宽，例如云盘厂商指定的最大 I/O 带宽。
++ 默认值："0MB"
+
+### `mode`
+
++ 确定哪些类型的 I/O 操作被计数并受 `max-bytes-per-sec` 阈值的限流。当前 TiKV 只支持 write-only 只读模式。
```
Comment (qiancai) [inline tikv-configuration-file.md:312]: "当前 TiKV 只支持 write-only 只读模式。" 是 "当前 TiKV 只支持 write-only 模式。"？

**Pull request merged**